tests/JsonSchemaMapper.Tests/TestTypes.cs

```diff
@@ -61,21 +62,29 @@ public static IEnumerable<ITestData> GetTestDataCore()
         yield return new TestData<DateTimeOffset>(
             Value: new(new DateTime(2021, 1, 1), TimeSpan.Zero), 
             AdditionalValues: [DateTimeOffset.MinValue, DateTimeOffset.MaxValue],
-            ExpectedJsonSchema: """{"type":"string","format": "date-time"}""");
+			// `date-time` format is RFC3339; not sure about DTO, but DT serializes as IS08601.
```

Comment:
Thanks. For similar reasons we replaced `"format" : "time"` with a pattern keyword for the case of `TimeSpan`. We might want to do the same here.

cc @eerhardt